apps/client/src/common/utils/eventsManager.ts

```diff
@@ -26,6 +26,6 @@ export const cloneEvent = (event: OntimeEvent, after?: string): ClonedEvent => {
     revision: 0,
     timeWarning: event.timeWarning,
     timeDanger: event.timeDanger,
-    custom: {},
+    custom: event.custom,
```

Comment:
@cpvalente 
was there a reason for not copying the custom data in the `cloneEvent()` funciton?

Reply:
Yes, in the workflows that we use to guide, the custom data belongs to the team, not to the editor or the schedule

In that light, it makes no sense to copy them into a new event

I am sure there are cases where this is not true, I guess we could make a small popup that let's you select the attributes that get copied. Until then I believe we should keep it simple

Reply:
hmmm yes I would very much like to be able to "deeply" clone an event

I have seen some apps call it it clone and duplicate
where duplicate auto increments addresses and so on
and clone is a straight clone

what do you think of that?
or should we rather go with an popup asking to keep or discard custom data?

Reply:
I am unsure we would have the clarity to modify the user data the way you describe

It wouldn't mean that we have an understanding of how the user is leveraging its data and can make safe assumptions over how to modify it.

I unfortunately don't think this is the case

In the case of cloning, from what i understood, copying the custom fields have the user more cleanup than the value it added. This was especially true before when the editor couldn't even see the custom fields. Again, custom fields are in the domain of operators using the cuesheet and operator view.

Having the popup for asking which fields to clone could be good, but it also has the downside of making the operations slower by having more steps

For now, we should focus on v3 and try not get too distracted and risk introducing bugs in parallel workflows